### PR TITLE
Unit tests for builder/dockerfile/support

### PR DESCRIPTION
**Pull request by tkopczynski**
Adding a few test cases for builder/dockerfile/support.go

Part of resolving #20784

Signed-off-by: Tomasz Kopczynski tomek@kopczynski.net.pl

### CODE REVIEW

File: builder/dockerfile/support_test.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package dockerfile
+
+import "testing"
+
+type TestCase struct {
```
Comment (calavera) [inline builder/dockerfile/support_test.go]: this struct doesn't need to be exported, does it?

**Comment by tkopczynski** [comment]
@calavera Yes, it doesn't need to be :) Changed that

**Comment by cpuguy83** [comment]
Could you document the utility of the `handleJSONArgs` function?

**Comment by tkopczynski** [comment]
@cpuguy83 I've added the docs, ptal

**Comment by cpuguy83** [comment]
:heart: Thanks!
LGTM

**Comment by vdemeester** [comment]
LGTM 🐮

**Pull request merged**